src/Model/Contact.php

```diff
@@ -1113,7 +1113,7 @@ public static function getIdForURL($url, $uid = 0, $update = null, $default = []
 			$contact = self::selectFirst(['id'], ['nurl' => $urls, 'uid' => $uid]);
 			if (!empty($contact['id'])) {
 				$contact_id = $contact['id'];
-				Logger::info('Fetched id by url', ['cid' => $contact_id, 'uid' => $uid, 'url' => $url, 'probed_url' => $data['url'], 'alias' => $data['alias'], 'addr' => $data['addr']]);
+				Logger::info('Fetched id by url', ['cid' => $contact_id, 'uid' => $uid, 'url' => $url, 'data' => $data]);
```

Comment:
This had been split into several fields because the `data` array is really large and this clogges the log file.

Reply:
Fair enough, I changed it.